js/ui/control/logo_control.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const DOM = require('../../util/dom');
+
+/**
+ * A `LogoControl` is a control that adds the Mapbox watermark
+ * to the map as required by the [terms of service](https://www.mapbox.com/tos/) for Mapbox
+ * vector tiles and core styles.
+ *
+ * @implements {IControl}
+**/
+
+class LogoControl {
+    constructor() {
+    }
+
+    onAdd(map) {
+        this._map = map;
```

Comment:
I guess we don't! I was following the `IControl` docs, but you're right it isn't necessary in this case.